eval/adjust.rkt

```diff
@@ -379,19 +388,20 @@
      (list (- slack (minlog z)))]
 
     [(ival-pow2)
+     ; same as multiplication
      (define x (first srcs))
      (list (logspan x))]
     
     ; TODO
     [(ival-erfc ival-erf ival-lgamma ival-tgamma ival-asinh ival-logb)
      (list (get-slack))]
+    ; TODO
     [(ival-ceil ival-floor ival-rint ival-round ival-trunc)
      (list (get-slack))]
     [else (map (const 0) srcs)]))        ; exponents for argumetns
 
 (define (get-slack)
```

Comment:
This has to be changed later because we give a user freedom with `(*rival-max-iterations*)`, therefore with iterations more than 5 it will crash

Reply:
Agreed—not essential right now but would be good to do.